sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/WebSocketsProxyConnectionHandler.java

```diff
@@ -39,10 +39,23 @@
 public class WebSocketsProxyConnectionHandler extends WebSocketsConnectionHandler {
     private static final String HTTPS_URI_FORMAT = "https://%s:%s";
 
-    private final InetSocketAddress connectionHostname;
+    private final InetSocketAddress proxyHostAddress;
     private final ProxyOptions proxyOptions;
     private final String fullyQualifiedNamespace;
-    private final String amqpBrokerHostname;
+    /**
+     * The value of 'host-name:port' field for the 'HTTP CONNECT host-name:port HTTP/1.1'
+     * request to the Proxy.
+     * e.g.
+     *   CONNECT &lt;eventubs-namespace&gt;.servicebus.windows.net:443 HTTP/1.1 <br/>
```

Comment:
does `<br/>` render correctly?

Reply:
It looks like we don't generate Javadoc for impl classes, but the newlines render correctly in editor hover.